pom.xml

```diff
@@ -297,50 +190,32 @@
         <dependency>
             <groupId>org.codehaus.groovy</groupId>
             <artifactId>groovy-all</artifactId>
-            <version>2.4.0</version>
             <classifier>indy</classifier>
```

Comment:
is this classifier required here given that it's in the parent? will it not inherit the classifier tag?